src/components/pages/rules/flyouts/ruleEditor/ruleEditor.js

```diff
@@ -181,157 +203,153 @@ export class RuleEditor extends LinkedComponent {
+                  <FormControl
+                    type='text'
+                    placeholder={t('rules.flyouts.ruleEditor.condition.valuePlaceholder')}
+                    link={condition.valueLink} />
                 </FormGroup>
+                {
+                  conditionLinks.length > 1 &&
+                  <Btn className='padded-top' svg={svgs.trash} onClick={this.deleteCondition(idx)}>{t('rules.flyouts.ruleEditor.delete')}</Btn>
+                }
               </Section.Content>
             </Section.Container>
-            <Section.Container collapsable={false}>
-              <Section.Content className='devices-affected'>
-                <div className='devices-affected-dynamic'>{this.state.devicesAffected}</div>
-                <div className='devices-affected-static'>{t('rules.flyouts.ruleEditor.devicesAffected')}</div>
-              </Section.Content>
-            </Section.Container>
-            <BtnToolbar className='apply-cancel'>
-              <Btn type='submit'>{t('rules.flyouts.ruleEditor.apply')}</Btn>
-              <Btn onClick={onClose}>{t('rules.flyouts.ruleEditor.cancel')}</Btn>
-            </BtnToolbar>
-          </form>
-        </Flyout.Content>
-      </Flyout.Container>
+          ))
+        }
+        <Section.Container collapsable={false}>
+          <Section.Content>
+            <FormGroup className='padded-top'>
+              <FormLabel>{t('rules.flyouts.ruleEditor.severityLevel')}</FormLabel>
+              <Radio
+                link={this.severityLink}
+                value={severityLevels[0]}>
+                <SeverityRenderer value={severityLevels[0]} context={{ t }} />
+              </Radio>
+              <Radio
+                link={this.severityLink}
+                value={severityLevels[1]}>
+                <SeverityRenderer value={severityLevels[1]} context={{ t }} />
+              </Radio>
+              <Radio
+                link={this.severityLink}
+                value={severityLevels[2]}>
+                <SeverityRenderer value={severityLevels[2]} context={{ t }} />
+              </Radio>
+            </FormGroup>
+          </Section.Content>
+          <Section.Content>
+            <FormGroup>
+              <FormLabel>{t('rules.flyouts.ruleEditor.ruleStatus')}</FormLabel>
+              <ToggleBtn
+                value={formData.enabled}
+                onChange={this.onToggle} >
+                {formData.enabled ? t('rules.flyouts.ruleEditor.ruleEnabled') : t('rules.flyouts.ruleEditor.ruleDisabled')}
+              </ToggleBtn>
+            </FormGroup>
+          </Section.Content>
+        </Section.Container>
+        <SummarySection>
+          <SummaryCount>{devicesAffected}</SummaryCount>
+          <SectionDesc>{t('rules.flyouts.ruleEditor.devicesAffected')}</SectionDesc>
+        </SummarySection>
+        {
+          error &&
+          <AjaxError className="rule-error">{error}</AjaxError>
```

Comment:
Is this the "rule-error" className defined somewhere? If not it doesn't need to be here.

Also, this is not the correct usage of the AjaxError method, see below.
```
{
  error && <AjaxError t={t} error={error} />
}
```